### PR TITLE
Adding lemmatizer functionality

### PR DESCRIPTION
**Pull request by Daanielvb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@ venv
 *.pyc
 *.swp
 .coverage
+
+#mac
+.DS_Store
```
Comment (marlomjobsom) [inline .gitignore]: I would suggest you to move this to the "# Folders" section

Comment (Daanielvb) [reply .gitignore]: done

File: core/rest/nltk_rest.py [hunk 2]
```diff
@@ -51,3 +51,12 @@ def words_snowball_stemmer():
         """
         words = request.data
         return NltkBusiness.words_snowball_stemmer(words)
+
+    @staticmethod
+    @rest_auth_nltk
+    def words_lemmatize():
```
Comment (marlomjobsom) [inline core/rest/nltk_rest.py]: How about to keep the same BO method name (lemmatize) instead?

Comment (Daanielvb) [reply core/rest/nltk_rest.py]: done

File: init.py [hunk 3]
```diff
@@ -46,6 +46,7 @@ def __set_nltk_end_points(flask_app_wrapper):
     """
     flask_app_wrapper.add_post_endpoint(['words_tokenize'], NltkRest.words_tokenize)
     flask_app_wrapper.add_post_endpoint(['words_snowball_stemmer'], NltkRest.words_snowball_stemmer)
+    flask_app_wrapper.add_post_endpoint(['words_lemmatize'], NltkRest.words_lemmatize)
```
Comment (marlomjobsom) [inline init.py]: How about to set the path to the "lemmatize" only?
I've also suggest you in other comment to update the method name as well

Comment (Daanielvb) [reply init.py]: done

File: tests/test_nltk_rest.py [hunk 4]
```diff
@@ -136,5 +136,37 @@ def test_words_snowball_stemmer_no_auth_content(self):
         self.assertEqual(b'<h1>Authentication Required!</h1>', response.data)
 
 
+class RequestWordsLemmatizerTestCase(NltkRestTestCase):
```
Comment (marlomjobsom) [inline tests/test_nltk_rest.py]: Can we update this classe name to RequestLemmatizerTestCase

Comment (Daanielvb) [reply tests/test_nltk_rest.py]: done

**Pull request merged**